literature.bib

```diff
@@ -178,6 +178,15 @@ @String{PhD
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Unpublished Entries %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@misc{SBK+:ICST25,    
+	title = {{Poster: Quantification of Feature-Interaction Masking in JHipster}},
+	author = {Schmidt, Tim Jannik and B{\"{o}}hm, Sabrina and Krieter, Sebastian and Th{\"u}m, Thomas and Acher, Mathieu},
+	howpublished = Poster,
+	month = MAR,
+	year = 2025,
+	note = ToAppear#{Poster at International Conference on Software Testing, Verification and Validation}
```

Comment:
Is this correct syntax? Not familiar with this.

Reply:
I think so but I am not sure either. @skrieter help!

Reply:
yes, that is correct

Reply:
One further note: As # is just the concatenation, it should be something like `ToAppear#{, Poster ...` to be shown correctly.